demo/admin/src/common/MasterMenu.tsx

```diff
@@ -1,16 +1,37 @@
 import { Menu, MenuCollapsibleItem, MenuContext, MenuItemRouterLink, useWindowSize } from "@comet/admin";
-import { Assets, Dashboard, Data, PageTree, Snips, Wrench } from "@comet/admin-icons";
+import { CurrentUser, useCurrentUser } from "@comet/cms-admin";
 import * as React from "react";
-import { useIntl } from "react-intl";
 import { useRouteMatch } from "react-router";
 
+import { Page } from "./pages/page.type";
+import { pages } from "./pages/pages";
+
 const permanentMenuMinWidth = 1024;
 
+const MenuItem = ({ basePath, page, user }: { basePath: string; page: Page; user: CurrentUser }) => {
```

Comment:
out of scope: this could be moved into the library?

Reply:
See https://github.com/vivid-planet/comet/pull/1223/commits/9b45903bc76070e2e4cd8d1c2ee108654286338a